### PR TITLE
Feature/replace header based pagination

### PR DESCRIPTION
**Pull request by ivo-ritense**

### CODE REVIEW

File: release-notes/major12/12.0.0/valtimo-backend-libraries.md [hunk 1]
```diff
@@ -18,6 +18,18 @@ The following features were added:
   The assignees of a user task are now saved in the database by their user ID. When implementations use keycloak, the
   assignee column from the task table in the databases is automatically migrated from email to user ID.
 
+* **Introduced replacement for header based pagination**
```
Comment (marijnritense) [inline release-notes/major12/12.0.0/valtimo-backend-libraries.md:21]:
```suggestion
* **Replacement for header based pagination**
```

File: release-notes/major12/12.0.0/valtimo-backend-libraries.md [hunk 2]
```diff
@@ -18,6 +18,18 @@ The following features were added:
...
+  
+  Several endpoints that use URLs in the HTTP headers to paginate results have a new version with endpoints that return
+  the pagination information in the response body. The following endpoints are impacted by this change:
+  
+  *  `/api/v1/choice-fields` -> `/api/v2/choice-fields`
```
Comment (marijnritense) [inline release-notes/major12/12.0.0/valtimo-backend-libraries.md:26]: Can you change this to a table?

File: release-notes/major12/12.0.0/valtimo-backend-libraries.md (hunk 1 above)
Comment (ivo-ritense) [reply release-notes/major12/12.0.0/valtimo-backend-libraries.md]: Fixed

File: release-notes/major12/12.0.0/valtimo-backend-libraries.md (hunk 2 above)
Comment (ivo-ritense) [reply release-notes/major12/12.0.0/valtimo-backend-libraries.md]: Fixed

**Review by marijnritense — APPROVED** [review]

**Pull request merged**